app/about_us/OurPartners.js

```diff
@@ -0,0 +1,51 @@
+import styles from "./partners.module.css";
+
+const OurPartners = () => {
+
+const CompanyName = [
+
+{company: "Alphabet",
+ Url: "./business_partners/alphabet-logo.png",
```

Comment:
```suggestion
 url: "./business_partners/alphabet-logo.png",
```
PascalCase variables are reserved for components in React.